src/Symfony/Component/Mime/Address.php

```diff
@@ -91,9 +91,9 @@ public function getEncodedName(): string
     }
 
     /**
-     * @param Address|string $address
+     * @param string|Address $address
```

Comment:
can be removed (more broadly: please remove all annotations that don't provide value anymore)